etc/grafana/dashboards/overview.json

```diff
@@ -7959,7 +7959,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "max(max_over_time(reth_rpc_server_calls_time_seconds[$__rate_interval])) by (method) > 0",
+          "expr": "max(max_over_time(reth_rpc_server_calls_time_seconds{instance=~\"$instance\"}[$__rate_interval])) by (method) > 0",
```

Comment:
any reason for using `=~` here?

Reply:
I don't want the results of other instances to affect the corresponding dashboard, such as having instanceA and instanceB, when checking the rpc performance of instanceA, I don't want the results of instanceB to affect it.

Reply:
sure I got that, was asking why `=~` instead of precise `=`?

Reply:
sorry for that, I don't get it first :). `=~` selects the label with regex matching, so it supports multi-instance values. Currently we select the instance for one value, if next time we need to support multiple values, this query expr(`=~`) doesn't need to modify 

<img width="714" alt="image" src="https://github.com/paradigmxyz/reth/assets/3627395/0c804fe2-d207-46e7-8104-57a1c996bb41">

Reply:
yeah sure, was wondering more specifically, why we could possibly select more than one instance here in the future? didn't come up with a good reason myself

Reply:
to me, I want to calculate the total value of the selected instances, so the regex filter will be helpful. 

btw, if we want to support multi instances, some expr need to adjust, eg `by (method)` -> `by (instance, method)`

Reply:
`by (method) -> by (instance, method)`, this would be a good issue @jsvisa